src/split-tests.js

```diff
@@ -58,14 +58,27 @@ export default function split_tests_init() {
                 for (let replacement of variant.content) {
                     let targets = document.querySelectorAll(replacement.selector);
                     for (let target of targets) {
-                        if (replacement.search && target.innerText.trim() != replacement.search.trim()) {
+                        if (replacement.search != '' && target.innerText.trim() != replacement.search.trim()) {
                             continue;
                         }
                         target.innerText = replacement.replace;
                     }
                 }
             }
 
+            if (variant.classes) {
+                for (let change of variant.classes) {
+                    let targets = document.querySelectorAll(change.selector);
+                    for (let target of targets) {
+                        if (change.change == 'add') {
+                            target.classList.add(change.class);
```

Comment:
I'm missing something - wouldn't this code mean that the added/removed classes were _always_ added/removed?

Reply:
Well it'll happen every time the variant that lands on the page with a class add/removal, but that will only happen some of the time. fwiw DomTests.php:76 is where that random selection happens.

Reply:
got it, thanks!